### DISCUSSION TITLE
Question: how to use  packages on mybinder?

### DISCUSSION THREAD

**Discussion by sje30** [fonsp/Pluto.jl#1419] [Questions about using Pluto]
Hello,

sorry for the obvious question, but we have been making some pluto notebooks.  those without any external packages work fine on mybinder, e.g.

https://sje30.github.io/catam-julia/casestudies/Permutations/permutationsnotebook.html

will launch on mybinder (clicking top right)

but this one, that uses extra packages, like Plots, does not work.  What are we missing?  Do we need to tell mybinder which extra packages to install?

https://sje30.github.io/catam-julia/casestudies/randomwalks/randomwalksnotebook.html

many thanks, Stephen

CC: @jmb280cam

**Comment by fonsp** [comment]
You wrote that it "does not work". Can you be more specific? What do you see?
